VERSION.txt

```diff
@@ -1 +1 @@
-24.4.4
+25.0.0
```

Comment:
If this is a major (breaking) change then you'd need to update the changelog - but I don't think it breaks anything, so the new version should be `24.5.0`.